sanowret-crystal.lic

```diff
@@ -14,6 +14,7 @@ class SanowretCrystal
     @invalid_rooms = []
     @worn_crystal = false
     args = parse_args(arg_definitions)
+    @noun = args.sanowret_noun ? args.sanowret_noun : 'sanowret'
```

Comment:
❌ This arg needs to be added to `arg_definitions` on line 9

Reply:
Oh, my mistake.  I was tired when I was doing this.

It was supposed to be from get_settings, not args.